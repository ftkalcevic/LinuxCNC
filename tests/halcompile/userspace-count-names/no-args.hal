loadusr -W userspace_count_names
list pin
