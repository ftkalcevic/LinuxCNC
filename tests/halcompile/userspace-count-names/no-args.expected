count=0
argc=1
argv[0]=userspace_count_names
argv[1]=(null)
userspace-count-names.0.out

